enterprise/models.py

```diff
@@ -107,3 +110,56 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+def logo_path(instance, filename):
+    """
+    Delete the file if it already exist and returns the enterprise customer logo image path.
+
+    :param instance: EnterpriseCustomerBrandingConfiguration object
+    :param filename: file to upload
+    :return path: path of image file e.g. enterprise_logo/1/filename
+    """
+    name = os.path.join('enterprise_logo', str(instance.id), filename)
+    fullname = os.path.join(settings.MEDIA_ROOT, name)
+    if os.path.exists(fullname):
+        os.remove(fullname)
+    return name
+
+
+class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):
+    """
+    Model that keeps track of enterprise branding configurations e.g. enterprise customer logo.
+
+    Fields:
+        enterprise_customer (ForeignKey[EnterpriseCustomer]): enterprise customer
+        logo (ImageField): enterprise customer image
+    """
+
+    enterprise_customer = models.OneToOneField(
+        EnterpriseCustomer,
+        blank=False,
+        null=False,
+        related_name='branding_configuration'
+    )
+    logo = models.ImageField(
+        upload_to=logo_path,
```

Comment:
Okay @mattdrayer. As, currently Edx-Enterprise is utilizing the storage mechanism of edx-platform such as `FileSystemStorage` as implemented in common.py and on production, edx-platform uploading the assets on AWS S3 as commented above.

If you are saying that enterprise application should have its own storage settings under [edx-enterprise repo](https://github.com/edx/edx-enterprise) then I want to know how can we **restrict/configure** the enterprise application to store its media data on its own S3 bucket instead of using `FileSystemStorage` of edx-platform ?